terra_layer/management/commands/update_postgis_source.py

```diff
@@ -0,0 +1,176 @@
+from django.core.management.base import BaseCommand
+
+from django_geosource.models import CSVSource
+from terra_layer.models import Layer
+
+fields_matches = {
+    "id": "ID",
```

Comment:
This is specific code in generic package, remove it PLS.

Reply:
It has been removed.